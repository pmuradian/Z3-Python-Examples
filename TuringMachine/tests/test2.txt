3 2 2
a b c
i f
i
f
a i a i 1
b i a f 0
4
a b a a
