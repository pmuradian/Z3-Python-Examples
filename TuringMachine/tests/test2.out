YES

a|i b|  a|  a|
a|  b|i a|  a|
a|  a|f a|  a|
YESa|i b| a| a| a| b|i a| a| a| a|f a| a| 